Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -110,6 +110,13 @@ enum ConfigCLI {
       }
     } catch let error as Config.Error {
       logMessage(.error, error)
+      logMessage(.info, """
```

Comment:
Not sure why `.error` was changed to `.info` here… tbh I don't remember if this only impacts the way it's presented (red vs yellow) or if it also impacts the exit code (w should still exit with non-zero in those cases)?

Reply:
Ah I thought since it was an info message, it should be `.info`, but good point, I'll change it to `.error`.